<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskForm</class>
 <widget class="QWidget" name="TaskForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>248</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>371</width>
     <height>230</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="TaskForm_task">
      <property name="text">
       <string>Задание</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="TaskForm_variant">
      <property name="text">
       <string>Вариант</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="TaskForm_answer">
      <property name="text">
       <string>Ваш ответ</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="TaskForm_enterans">
      <property name="plainText">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="TaskForm_sendans">
      <property name="text">
       <string>Отправить ответ</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="TaskForm_exit">
      <property name="text">
       <string>Выйти</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>TaskForm_enterans</tabstop>
  <tabstop>TaskForm_sendans</tabstop>
  <tabstop>TaskForm_exit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
